packages/stylelint-config/test/scss-valid.scss

```diff
@@ -22,8 +22,10 @@ $map: (
 	display: inline-block;
 }
 
+// stylelint-disable no-invalid-position-at-import-rule
 @import "../some/url";
 @import "../another/url";
+// stylelint-enable no-invalid-position-at-import-rule
```

Comment:
With the addition of `packages/stylelint-config/test` in `.stylelintignore` in his PR at https://github.com/WordPress/gutenberg/pull/38091/files#diff-36ae73c99ebd1b4072a3164e014df72eb094703df4548dd51938aa7faaa7f8d2R1 are the ignore directives needed here?

Reply:
Yes, the `.stylelintignore` is only used when running `npm run lint-css` in the project root but not when running `npm run test-unit packages/stylelint-config` for the tests.

The rule is new in v5, see https://github.com/stylelint-scss/stylelint-config-recommended-scss/commit/1e40cc1e1a288cc01aeeeccd8a0f72968bc9807a. Instead of removing the lines I added the comments for now.

Reply:
Cool, makes sense